### PR TITLE
Make getABI_* calls in wrapper should be static

### PR DESCRIPTION
**Pull request by tonykwok1992**
### What does this PR do?
getABI_* call in wrapper should be static method. we do not need an instance of wrapper to encode function

### Where should the reviewer start?
All files

### Why is it needed?
we do not need an instance of wrapper to encode function

## Checklist

- [x] I've read the contribution guidelines.
- [x] I've added tests (if applicable).
- [ ] I've added a changelog entry if necessary.

### CODE REVIEW

File: codegen/src/test/resources/solidity/abifuncs/AbiFuncs.sol [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pragma solidity >= 0.8.7;
+
+contract AbiFuncs {
+
+
+    function hi(uint256 x) public {
```
Comment (NickSneo) [inline codegen/src/test/resources/solidity/abifuncs/AbiFuncs.sol:6]: This seems like an empty solidity file, can't we reuse any pre-existing .sol file for the test?

File: codegen/src/test/resources/solidity/abifuncs/build/AbiFuncs.abi [hunk 2]
```diff
@@ -0,0 +1 @@
+[{"inputs":[{"internalType":"uint256","name":"x","type":"uint256"}],"name":"hi","outputs":[],"stateMutability":"nonpayable","type":"function"}]
```
Comment (NickSneo) [inline codegen/src/test/resources/solidity/abifuncs/build/AbiFuncs.abi:1]: Same lets use any existing solidity and compile .abi and .bin files

**Review by NickSneo — CHANGES_REQUESTED** [review]
Instead of adding a new empty solidity file for testing, lets reuse any existing .sol file

File: codegen/src/test/resources/solidity/abifuncs/AbiFuncs.sol (hunk 1 above)
Comment (tonyinf) [reply codegen/src/test/resources/solidity/abifuncs/AbiFuncs.sol:6]: We can, but we can only test successful compilation if I use existing files, but cannot test java file output, if I modify the existing java file to add abiFuncs, we lose the test case of abiFuncs = false

Comment (tonykwok1992) [reply codegen/src/test/resources/solidity/abifuncs/AbiFuncs.sol:6]: I also want to test java file output matches, if I use any existing file, I will have to modify those java files to add getABI* calls, but that will actually invalidate the test case with abiFunc=false

Comment (NickSneo) [reply codegen/src/test/resources/solidity/abifuncs/AbiFuncs.sol:6]: Okay got it

**Review by NickSneo — APPROVED** [review]
LGTM

**Pull request merged**